provisioning/Samples/service/EnrollmentSample/EnrollmentSample.cs

```diff
@@ -42,40 +42,39 @@ public async Task RunSampleAsync()
         public async Task QueryIndividualEnrollmentsAsync()
         {
             Console.WriteLine("\nCreating a query for enrollments...");
-            QuerySpecification querySpecification = new QuerySpecification("SELECT * FROM enrollments");
-            using (Query query = _provisioningServiceClient.CreateIndividualEnrollmentQuery(querySpecification))
+            var querySpecification = new QuerySpecification("SELECT * FROM enrollments");
+            using Query query = _provisioningServiceClient.CreateIndividualEnrollmentQuery(querySpecification);
+            while (query.HasNext())
             {
-                while (query.HasNext())
-                {
-                    Console.WriteLine("\nQuerying the next enrollments...");
-                    QueryResult queryResult = await query.NextAsync().ConfigureAwait(false);
-                    Console.WriteLine(queryResult);
-                }
+                Console.WriteLine("\nQuerying the next enrollments...");
+                QueryResult queryResult = await query.NextAsync().ConfigureAwait(false);
+                Console.WriteLine(queryResult);
             }
         }
 
         public async Task CreateIndividualEnrollmentTpmAsync()
         {
             Console.WriteLine("\nCreating a new individualEnrollment...");
             Attestation attestation = new TpmAttestation(TpmEndorsementKey);
-            IndividualEnrollment individualEnrollment =
+            var individualEnrollment =
                     new IndividualEnrollment(
                             RegistrationId,
-                            attestation);
-
-            // The following parameters are optional:
-            individualEnrollment.DeviceId = OptionalDeviceId;
-            individualEnrollment.ProvisioningStatus = OptionalProvisioningStatus;
-            individualEnrollment.InitialTwinState = new TwinState(
+                            attestation)
+                    {
+                        // The following parameters are optional:
+                        DeviceId = OptionalDeviceId,
+                        ProvisioningStatus = OptionalProvisioningStatus,
+                        InitialTwinState = new TwinState(
                 null,
                 new TwinCollection()
                 {
                     ["Brand"] = "Contoso",
                     ["Model"] = "SSC4",
                     ["Color"] = "White",
-                });
-            individualEnrollment.Capabilities = OptionalEdgeCapabilityEnabled;
-            
+                }),
+                        Capabilities = _optionalEdgeCapabilityEnabled
+                    };
+

```

Comment:
I prefer what we used to do here instead of using nested blocks to make it not easy to read.